src/core/client/ui/shared/typography.css

```diff
@@ -127,11 +127,29 @@
   letter-spacing: calc(0.2em / 16);
 }
 
-.textField {
+.validationMessage {
```

Comment:
Just synced with @IAmSamHankins, and we want to keep the name `alertMessage` as defined in the designs.

Reply:
only the css name, not the component name.